site/app/entities/course/CourseMaterial.php

```diff
@@ -118,6 +148,22 @@ public function getReleaseDate(): \DateTime {
         return $this->release_date;
     }
 
+    public function getUploadedBy(): string {
+        return $this->uploaded_by;
+    }
+
+    public function getUploadDate(): \DateTime {
+        return $this->upload_date;
+    }
+
+    public function getLastEditBy(): string {
+        return $this->last_edit_by;
+    }
+
+    public function getLastEditDate(): \DateTime {
+        return $this->last_edit_date;
+    }
```

Comment:
All of these could return null, right?

Reply:
Yeah.

Reply:
Null should be added to the return type then.

Reply:
It can be either of two types: DateTime or Null. Should I add both?

Reply:
@williamjallen any update on it?

Reply:
You can mark the return type as a nullable DateTime with `?DateTime`.  Alternatively, you could use an equivalent union type like `DateTime|null`.